views/layouts/default.hbs

```diff
@@ -42,21 +42,21 @@
     <footer id="footer">
       <div class="section-wrapper">
         <div class="third">
-          <a class="footer-logo-wrapper" href="https://www.mozilla.org" target="_blank" rel="noopener"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
+          <a class="footer-logo-wrapper" href="https://www.mozilla.org?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=footer-link" target="_blank" rel="noopener"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
```

Comment:
Unrelated to this, but I just noticed seeing them side-by-side(tm), but why is it `{{ SERVER_URL }}` here, but `{{ serverUrl }}` above (see ./views/email/email_partials/email_footer.hbs:L17):

```html
<img src="{{ serverUrl }}/img/moz-bw-email.png" width="40" height="40" style="width: 40px; height: 40px;" />
```

Reply:
Good point - we should try to consolidate on the naming convention. I can update `controllers/user.js` to set the email context var to be `SERVER_URL` like we set the website context var.